Psychtoolbox/PsychProbability/Shuffle.m

```diff
@@ -1,4 +1,4 @@
-function [Y,index] = Shuffle(X)
+function [Y,index] = Shuffle(X, bindDim)
 % [Y,index] = Shuffle(X)
```

Comment:
Here you need to rewrite to % [Y, index] = Shuffle(X [, bindDim]); so users know the bindDim argument can be provided, but is optional.